doc/customize.md

```diff
@@ -0,0 +1,449 @@
+in this example that leverages the JSON config manager from
+`traitlets`:
+
+```python
+from traitlets.config.manager import BaseJSONConfigManager
+path = "/home/damian/miniconda3/envs/rise_latest/etc/jupyter/nbconfig"
+cm = BaseJSONConfigManager(config_dir=path)
+cm.update("rise", {
+              "theme": "sky",
+              "transition": "zoom",
+              "start_slideshow_at": "selected",
+})
+```
+
+**Notes:**
+
+* `path` is where the `nbconfig` is located. This will vary depending
+  on where you "installed" and "enabled" the nbextension.
+
+* you can use `jupyter --paths` to see the path locations that are
+  applicable.
+
+* for more information, see these docs:
+  * <http://jupyter.readthedocs.io/en/latest/projects/jupyter-directories.html>
+  * <http://jupyter-notebook.readthedocs.io/en/latest/frontend_config.html>.
+
+
+## Keyboard shortcuts and Jupyter actions
+
+Here are the Jupyter actions registered by RISE:
+
+    action name             key      behaviour
+    ------------------------------------------------------
+    RISE:slideshow         alt-r  enter/exit RISE Slideshow
+    RISE:smart-exec               execute cell, move to the next if on same slide
+    RISE:toggle-slide     shift-i (un)set current cell as a Slide cell
+    RISE:toggle-subslide  shift-u (un)set current cell as a Sub-slide cell
+    RISE:toggle-fragment  shift-f (un)set current cell as a Fragment cell
+    RISE:toggle-note              (un)set current cell as a Note cell
+    RISE:toggle-skip              (un)set current cell as a Skip cell
+    RISE:render-all-cells         render all cells (all cells go to command mode)
+    RISE:edit-all-cells           edit all cells (all cells go to edit mode)
+
+Some, but not all, come bound to default keyboard shortcuts. There are
+2 ways you can change the bindings
+
+### Through JSON
+
+Like the other settings described in this section, you can define
+shortcuts in JSON with e.g.
+
+    {
+     ...
+     "rise": {
+         "shortcuts": {
+             "slideshow": "alt-a",
+             "edit-all-cells": "ctrl-e"
+         }
+    }
+
+With the above settings, RISE would **not** bind the default `Alt-R`
+key to `RISE:slideshow`, but it would bind `Alt-A` instead. It would
+also bind `RISE:edit-all-cells` to `Ctrl-e`.
+
+### Through `custom.js`
+
+You can also use these actions in some regular javascript code,
+typically your `~/.jupyter/custom/custom.js`. Here is an example that
+will attach one of these actions to a custom keyboard shortcut:
+
+```javascript
+    define(
+        ['base/js/namespace'],
+        function(Jupyter) {
+
+            let command_shortcuts = Jupyter.keyboard_manager.command_shortcuts;
+
+            // set / unset the 'Slide' tag in slideshow metadata
+            command_shortcuts.set_shortcut(
+                'alt-a', 'RISE:slideshow');
+        })
+```
+
+Note that with this approach, you will end up with the
+`RISE:slideshow` action bound to **both** `Alt-R` and `Alt-A`.
```

Comment:
Why is that? If you can remember it for me :wink:

Reply:
well if you use the keyboard manager directly, RISE cannot be aware of that, and so it will still define the shortcut as defined in your JSON config.

Reply:
Got it!